lms/djangoapps/verify_student/models.py

```diff
@@ -721,8 +721,8 @@ def submit(self, copy_id_photo_from=None):
                 self.status = "must_retry"
                 self.error_msg = response.text
                 self.save()
-        except Exception as error:
-            log.exception(error)
+        except Exception:
+            log.exception("Request sending to Software Secure failed, setting status to must_retry")
```

Comment:
@awaisdar001 ok I am going to add username, in message, of a user for which request failed.